config/config.go

```diff
@@ -353,6 +353,14 @@ type (
 		RangeBloomFilterNumHash uint64 `yaml:"rangeBloomFilterNumHash"`
 	}
 
+	// Tracer is the config for tracer
+	Tracer struct {
```

Comment:
move this struct to /pkg/tracer

Reply:
also, we need to specify a default endpoint? like `iotex.status.me/tracing` or `tracing.iotex.io`

Reply:
tracer `endpoint` leave blank(default value) will skip the integration of tracer, tracer does not sampling. maybe tracer is only used by us.